docs/docs/guides/computed-attributes.mdx

```diff
@@ -0,0 +1,265 @@
+        peer: LocationSite
+        optional: false
+        cardinality: one
+        kind: Attribute
+```
+
+:::note
+
+In your template, you can utilize most of the built-in filters provided by Jinja2!
+
+For more information, please consult the [Schema Reference](/reference).
+
+:::
+
+You can now load this schema into your Infrahub instance. For more details, refer to the [Schema Creation Guide](/guides/create-schema).
+
+## Adding Python computed attribute
+
+Creating a Python computed attribute is a two-step process:
+
+1. Add the attribute to the schema.
+2. Prepare a Python Transform to implement the logic.
+
+In the following example we will create a computed attribute `description` for our `InfraCircuit` node. The value will be a combination of all endpoints locations.
+
+### Add the attribute to the schema
+
+Here's the schema we'll use as a starting point:
+
+```yml
+---
+version: "1.0"
+nodes:
+  - name: Site
+    namespace: Location
+    display_labels:
+      - name__value
+    attributes:
+      - name: name
+        kind: Text
+        unique: true
+    relationships:
+      - name: circuit_endpoints
+        cardinality: many
+        optional: true
+        peer: InfraCircuitEndpoint
+        kind: Component
+
+  - name: Circuit
+    namespace: Infra
+    display_labels:
+      - circuit_id__value
+    attributes:
+      - name: circuit_id
+        kind: Text
+        unique: true
+      - name: computed_description
+        kind: Text
+        computed_attribute:
+          kind: TransformPython
+          transform: computed_circuit_description # Transform's name
+        read_only: true  # You must set the attribute to read-only as the value will be handled by the system.
+        optional: true   # As it's a Python kind of attribute it must be optional
+    relationships:
+      - name: endpoints
+        peer: InfraCircuitEndpoint
+        optional: true
+        cardinality: many
+        kind: Component
+
+  - name: CircuitEndpoint
+    namespace: Infra
+    label: Circuit endpoint
+    display_labels:
+      - name__value
+    attributes:
+      - name: name
+        kind: Text
+        unique: true
+    relationships:
+      - name: circuit
+        peer: InfraCircuit
+        cardinality: one
+        kind: Attribute
+      - name: location
+        peer: LocationSite
+        cardinality: one
+        kind: Attribute
+```
+
+Once completed, you can load this schema into Infrahub.
+
+:::note
+
+You can specify a transform that doesn't yet exist in Infrahub, attribute will just remain empty. Once the transform is loaded, Infrahub will automatically catch up and compute the value of this attribute.
+
+:::
+
+### Create the Python transform
+
+Please refer to the [Python transform guide](/guides/python-transform) for further details.
+
+1. First we prepare a GraphQL query that returns the data we need
+
+:::warning
+
+GraphQL query used to compute attribute's value requires `ID` as parameter.
+
+```graphql
```

Comment:
Not sure if this code block adds any value.

Reply:
It's really to emphasize that we need to put ID as parameter in this exact way, otherwise it won't work...